modules/veteran/app/controllers/veteran/v0/accredited_representatives_controller.rb

```diff
@@ -0,0 +1,154 @@
+# frozen_string_literal: true
+
+module Veteran
+  module V0
+    class AccreditedRepresentativesController < ApplicationController
+      skip_before_action :authenticate
+      before_action :feature_enabled
+      before_action :verify_type
+      before_action :verify_sort
+      before_action :verify_long
+      before_action :verify_lat
+
+      DEFAULT_PAGE = 1
+      DEFAULT_PER_PAGE = 10
+      DEFAULT_SORT = 'distance_asc'
+      PERMITTED_TYPES = %w[attorney claims_agent organization].freeze
+
+      PERMITTED_ORGANIZATION_SORTS = %w[distance_asc distance_desc name_asc name_desc].freeze
+      PERMITTED_REPRESENTATIVE_SORTS = %w[distance_asc distance_desc first_name_asc first_name_desc last_name_asc
+                                          last_name_desc].freeze
+      ORGANIZATION_SORT_MAP = {
+        'distance_asc' => '',
+        'distance_desc' => '',
+        'name_asc' => '',
+        'name_desc' => ''
+      }.freeze
+      def index
+        if @search_model_type == 'organization'
+          query_results = if search_params[:name]
+                            base_organizations_query.find_with_name_similar_to(search_params[:name])
+                          else
+                            base_organizations_query
+                          end
+          collection = Common::Collection.new(Veteran::Service::Organization, data: query_results)
+          resource = collection.paginate(**pagination_params)
+          render json: resource.data,
```

Comment:
I'm pretty sure that calling `render` twice will throw an error.

Are you wanting to send both `resource.data` and `query_results` as `JSON`? If so, they'll need to be combined.

Reply:
Good catch. That's a testing artifact that got uncommented before I pushed.